README.md

```diff
@@ -81,225 +169,72 @@ function fetchComments() {
 }
 ```
 
-### [X] Description of interaction can be optional
-
-If we do not specify description of interaction by `@pact-description`, the description is set from the name of function / variable / property.
-
-## [x] Specify REST method of interaction
+`@pact-response-status` - _optional_ - sets response status, if not provided, it is set based on given HTTP method
 
 ```ts
 /**
  * @pact
- * @pact-method GET
+ * @pact-response-status 200
  */
 function fetchComments() {
     // ...
 }
 ```
 
-## [x] Specify response status of interaction
+`@pact-path` - sets path
 
 ```ts
 /**
  * @pact
- * @pact-response-status 200
+ * @pact-path /api/images/100
  */
-function fetchComments() {
+function fetchImage(imageId: number) {
     // ...
 }
 ```
 
-### [X] Response status can be optional argument
-
-If we do not use `@pact-response-status`, the proper response status is set based on given HTTP method.
-
-## [x] Specify api path of interaction
+`@pact-request-header` - _optional if not defined in config file_ - adds header to current request, can override option defined in `pacts.config.js`
 
 ```ts
 /**
  * @pact
- * @pact-path /api/images/100
+ * @pact-request-header "Content-Type" "application/pdf"
  */
 function fetchImage(imageId: number) {
     // ...
 }
 ```
 
-## [x] Create response body in pacts from returned type of function
+`@pact-response-header` - _optional if not defined in config file_ - adds header to current response, can override option defined in `pacts.config.js`
 
 ```ts
 /**
  * @pact
+ * @pact-response-header "Content-Type" "application/pdf"
  */
-function fetchComments() {
+function fetchImage(imageId: number) {
     // ...
-    return data;
 }
 ```
 
-Pact-gen-ts recognizes response body type of interaction from returned object.
-
-## [x] Set manually what is response body
+`@pact-response-body` - sets expected body for current response, **IMPORTANT** - JSDoc has to be applied to separate variable - **not** directly to axios response
```

Comment:
I would write:
````
`@pact-response-body` - sets expected body for current response

```ts
/**
 * @pact
 */
async function fetchComments() {
    const response = await axios.get<string>('/api');
    /** @pact-response-body */
    const data = response.data;
    // ...
}
```
**IMPORTANT** - JSDoc has to be applied to separate variable - **not** directly to axios response
```ts
    /** @pact-response-body */ - WRONG!;
    const response = await axios.get<string>('/api');
    /** @pact-response-body */ - CORRECT;
    const data = response.data;
```
````

We should give firstly some valid example of this annotation, and then give some additional information